sql/parser/Makefile

```diff
@@ -1,5 +1,6 @@
-.PHONY: all
-all: sql.go keywords.go reserved_keywords.go
+TARGETS = sql.go keywords.go reserved_keywords.go
+.PHONY: all $(TARGETS)
```

Comment:
@petermattis The problem is that if you have a commit which has generated sql.go from sql.y incorrectly (e.g. wrong version of Go), "make" will not see a need to re-generate.

Given we only run "make" in sql/parser, or "go generate", by hand it seems reasonable to force re-generation at all times.